### PR TITLE
Updated ocr.c

### PR DESCRIPTION
**Pull request by Vaishnavi301201**
Added line nos. => 11, 660, 661, 673, 682 to fix the memory flow issue in ocr.c .

**In raising this pull request, I confirm the following (please check boxes):**

- [> ] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [ >] I have checked that another pull request for this purpose does not exist.
- [ >] I have considered, and confirmed that this submission will be valuable to others.
- [ >] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [ >] I give this submission freely, and claim no ownership to its content.
- [ ] **I have mentioned this change in the [changelog](https://github.com/CCExtractor/ccextractor/blob/master/docs/CHANGES.TXT).**

**My familiarity with the project is as follows (check one):**

- [ >] I have never used CCExtractor.
- [ ] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [ ] I am an active contributor to CCExtractor.

---

I have reviewed the code and added the changes to ocr.c so that the code will run smoothly.

### CODE REVIEW

File: src/lib_ccx/ocr.c [hunk 1]
```diff
@@ -659,6 +660,8 @@ char *ocr_bitmap(void *arg, png_color *palette, png_byte *alpha, unsigned char *
 						last_font_tag = font_tag;
 					}
 					last_font_tag_end = strstr(last_font_tag, ">");
+					if (last_font_tag_end > line_end)
+						last_font_tag_end = NULL;
```
Comment (cfsmp3) [inline src/lib_ccx/ocr.c:664]: Let's put this line in its own block ( { } )
I know most of our code doesn't do it but let's say single line branches without blocks are out of fashion these days to how prone they are to introduce bugs and security problems :-)

**Review by cfsmp3 — CHANGES_REQUESTED** [review]

File: src/lib_ccx/ocr.c (hunk 1 above)
Comment (Vaishnavi301201) [reply src/lib_ccx/ocr.c:664]: I'll try and make the changes and commit withnin the day :)

Comment (Vaishnavi301201) [reply src/lib_ccx/ocr.c:664]: I'll try and make the changes within the day :)

**Comment by cfsmp3** [comment]
This needs to be rebased - there's conflicts now
(note that the correct solution would have been to have everything on the same PR, but not really important - just rebase and I'll merge)

**Comment by cfsmp3** [comment]
Closing since there'a build errors and seems abandoned.

**Pull request closed without merging**